portfolio/src/main/webapp/script.js

```diff
@@ -48,9 +48,26 @@ function bubbleFall() {
     }
 }
 
-// Get message from the data servlet.
-function getMessage() {
-  fetch('/data').then(response => response.text()).then((message) => {
-    document.getElementById('message-container').innerText = message;
+// Get messages from the data servlet.
+function getMessages() {
+  fetch('/data').then(response => response.json()).then((messages) => {
+    console.log(messages);
+
+    // Find container.
+    const messageContainer = document.getElementById('message-container');
+
+    // Add each message to container as list element.
+    for (let i = 0; i < 3; i++) {
```

Comment:
Nice! Could also use for...of loop here to avoid having to hard-code 3.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of

Reply:
Thank you! I will definitely use that for the comment section.